apps/storybook/stories/Namekit/RegistrationPriceInfo.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  PrimaryRegistrationStatus,
+  SecondaryRegistrationStatus,
+  buildTimestamp,
+  now,
+} from "@namehash/ens-utils";
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+  RegistrationPriceInfo,
+  RegistrationPriceInfoDisplayingFormat,
+} from "@namehash/namekit-react";
+
+const ELEVEN_DAYS_FROM_NOW_TIMESTAMP = buildTimestamp(
+  BigInt(new Date().getTime()) / 1005n,
+);
+
+export const DisplayIconWithTooltip: Story = {
+  args: {
+    displayIconWithTooltip:
+      RegistrationPriceInfoDisplayingFormat.IconWithTooltip,
+    registration: {
+      expiryTimestamp: ELEVEN_DAYS_FROM_NOW_TIMESTAMP,
+      primaryStatus: PrimaryRegistrationStatus.Expired,
+      secondaryStatus: SecondaryRegistrationStatus.RecentlyReleased,
+      registrationTimestamp: now(),
+      expirationTimestamp: now(),
+    },
+    parsedName: {
+      namehash:
+        "0x54f4b08bc10ab9368020f2afdd830ab742058481586c4ae87ba6e37c8912a6ec",
+      slug: "expiredDomain.eth",
+      displayName: "expiredDomain.eth",
+      normalizedName: "expiredDomain.eth",
+      labelName: "expiredDomain",
+      labelHash:
+        "0x9458d05651ff8443f025747bf1a859bb26f2815f70d68e8cc9286027013961cd",
+      unwrappedTokenId:
+        113026375855800814118284943830039256141780833603018686252863399931967234376006n,
+      wrappedTokenId:
+        89103332435335212332047307598497361224309437240941958630163555473262172236931n,
+    },
+  },
+};
+export const DisplayTextAndIcon: Story = {
```

Comment:
In the output of this story the font styling of the price value appears to be heavier weight than all the other output text.

The weight of all the output text should match. The price shouldn't be heavier.

Reply:
I've also added a date tooltip to the period related displayed text